packages/gatsby-remark-prismjs/README.md

```diff
@@ -68,6 +68,12 @@ plugins: [
                 },
               },
             ],
+            // Customize the prompt used in shell output
+            // Leave empty to hide prompts
+            prompt: {
```

Comment:
I wonder if we should move this to the snippet itself instead. We do the same with line numbers and other plugins.
```
```bash prompt=root@localhost
```

```
```bash prompt={user:"root",host:"localhost"}
```

What do you think?

Reply:
I can see the idea behind it but I made this decision because

a) It reduces clutter in the snippet specification (esp. with other plugins)
b) As far as I can see it, it's something you either want on or off

But I'm happy to amend if you disagree with these points

Reply:
I'm working on adding the best of both worlds. So new logic will be:

- `prompt.global` setting defaulting to false (which will add prompts to every line)
- Including `{outputLines: <range>}` which will add prompts to any line _not_ in the range